google/api_core/bidi.py

```diff
@@ -91,8 +91,12 @@ def __init__(self, queue, period=1, initial_request=None):
     def _is_active(self):
         # Note: there is a possibility that this starts *before* the call
         # property is set. So we have to check if self.call is set before
-        # seeing if it's active.
-        return self.call is not None and self.call.is_active()
+        # seeing if it's active. We need to return True if self.call is None.
+        # See https://github.com/googleapis/python-api-core/issues/560.
+        if self.call is not None and not self.call.is_active():
```

Comment:
@parthea 

It would be simpler and much clearer to have `return self.call is None or self.call.is_active()`. This is equivalent to what you have, by DeMorgan's theorem.